assets/pages/contributions-landing/contributionsLanding.jsx

```diff
@@ -28,10 +28,21 @@ import ContributionAwarePaymentLogosContainer from './containers/contributionAwa
 import { createPageReducerFor } from './contributionsLandingReducer';
 
 
+// ----- Redux Store ----- //
+
+const countryGroupId: CountryGroupId = detect();
+
+const store = pageInit(createPageReducerFor(countryGroupId));
+
+const oneOffOnly = (store && store.getState().common.abParticipations.oneOffOnlyTest) || 'notintest';
+
 // ----- Internationalisation ----- //
 
 const defaultHeaderCopy = ['Help us deliver', 'the independent', 'journalism the', 'world needs'];
-const defaultContributeCopy = 'Make a monthly commitment to support The Guardian long term or a one-time contribution as and when you feel like it – choose the option that suits you best.';
+const defaultContributeCopy = oneOffOnly && oneOffOnly === 'oneOffOnly'
```

Comment:
Maybe to make it more readable I would rename `oneOffOnly` variable to be `oneOffOnlySelectedVariant`